builtin/aws/lambda/platform.go

```diff
@@ -829,6 +869,11 @@ type Config struct {
 	// Valid values are: "x86_64", "arm64"
 	// Defaults to "x86_64".
 	Architecture string `hcl:"architecture,optional"`
+
+	// The storage size (in MB) of the Lambda function's `/tmp` directory.
+	// Must be a value between 512 and 10240.
+	// Defaults to 512 MB.
+	Storage int `hcl:"storage,optional"`
```

Comment:
I recommend calling this `storagemb` so that it's obvious what unit Storage is without having to look it up in the Waypoint docs. We do something similar in Nomad for Memory:

https://tip.waypointproject.io/plugins/nomad#resources-memorymb

Reply:
Done! - https://github.com/hashicorp/waypoint/pull/3213/commits/f3c758f20a5647096e35fad636cd890379b87bb6